src/docs/references/transaction-process-actions/index.md

```diff
@@ -45,8 +45,70 @@ Initialize a new transaction from a listing.
 
 **Configuration options**: -
 
+### :action/privileged-set-line-items
+
+Enables custom pricing. Sets given line items and calculates totals for
+each line item and for the transaction.
```

Comment:
Suggestion: Instead of _Enables custom pricing._ let's write _Defines transaction price and breakdown._ or something of the sort. This action is now the default way to set the pricing for a marketplace so in that sense it does not enable any special feature but is used for the standard pricing in Flex.